core/type/pom.xml

```diff
@@ -64,5 +64,17 @@
             <version>${project.parent.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Why are these dependencies added?

Reply:
No idea, this seems to have happened automatically? Should I remove the dependency at line 67, or leave it since it seems to not break anything?

Reply:
Removed both instances of this dependency